drivers/SmartThings/lan-thing/src/init.lua

```diff
@@ -0,0 +1,12 @@
+local capabilities = require "st.capabilities"
+local Driver = require "st.driver"
+
+--should anything beyond refresh be added?
```

Comment:
nit - do we want to remove this comment?

Reply:
We can do it later in a commit to main. This is just the PR to move to beta. It copies the existing commit.